src/Feature/AbstractManager.ts

```diff
@@ -35,4 +38,107 @@ export default abstract class AbstractManager extends AbstractFeature<Feature> i
     protected abstract doUpdate(path: string): Promise<boolean>;
 
     protected abstract doRefresh(): Promise<{ [k: string]: boolean }>;
+
+    private fakeTitleElementMap = new Map<HTMLElement, HTMLElement>();
+
+    protected addFakeTitleElement(originalElement: HTMLElement, title: string): void {
```

Comment:
Abstract manager provides base logic for all managers.
This method is specific logic for specific managers.
To apply DRY in this case you can create something like FakeTitleElementService to create elements. But you will have to add specific events in specific cases because mouseLeaveHandler is excess for InnerTitleManager.